index.html

```diff
@@ -223,22 +234,28 @@
 Portions of the work on this specification have been funded by the United States
 Department of Homeland Security's (US DHS) Science and Technology Directorate
 under contracts HSHQDC-16-R00012-H-SB2016-1-002, and HSHQDC-17-C-00019, as well
-as  the US DHS Silicon Valley Innovation Program under contracts
-70RSAT20T00000010, 70RSAT20T00000029, 70RSAT20T00000030. The content of this specification does not
-necessarily reflect the position or the policy of the U.S. Government and no
-official endorsement should be inferred.
+as the US DHS Silicon Valley Innovation Program under contracts
+70RSAT20T00000010, 70RSAT20T00000029, 70RSAT20T00000030. The content of this
```

Comment:
```suggestion
70RSAT20T00000010, 70RSAT20T00000029, 70RSAT20T00000030, 70RSAT20T00000003, 70RSAT20T00000033. The content of this
```

Reply:
Github doesn't let me apply two changes to the same line. I've merged your numbers with @kdenhartog's numbers.